app/src/main/AndroidManifest.xml

```diff
@@ -96,8 +96,11 @@
         <activity
             android:name=".skills.feedback.FeedbackActivity"
             android:label="See All Reviews"
-            android:parentActivityName=".skills.feedback.FeedbackActivity"
-            android:theme="@style/PreferencesThemeLight" />
+            android:theme="@style/PreferencesThemeLight">
+            <meta-data
+                android:name="android.support.PARENT_ACTIVITY"
+                android:value="org.fossasia.susi.ai.skills.SkillsActivity" />
+        </activity>
```

Comment:
Revert

Reply:
Can you please elaborate.I am not able infer requested changes.

Reply:
Revert this change

Reply:
Could you please review this @iamareebjamal

Reply:
I already did. Requested changes have still not been made

Reply:
Still not reverted